ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type Interface interface {
```

Comment:
Looks like mockery has the `-inpkg` flag which will deal with this. E.g.:
```
mockery -name Interface -inpkg
```

OK, I'm good with `ringpop.Interface` :smile: